### DISCUSSION TITLE
GraphQL Enums

### DISCUSSION THREAD

**Discussion by reharik** [prisma/prisma#9425] [Q&A]
I have an enum in my prisma schema like this

```
enum UserStatus {
    INVITED
    ACTIVE
    DEACTIVATED
}

```
In my Graphql Schema I have an enum like this

```
enum UserStatus {
	INVITED
	ACTIVE
	DEACTIVATED
}
```

Now multiply this times ... X > 15 and growing.

The problem is clear, perhaps I'm doing something wrong?  But I can't see how to avoid this.
Thanks,
Raif

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
